lib/diagnostic.rb

```diff
@@ -37,9 +43,13 @@
 # of multiplying by two the result of adding `a` and `b`. Name the method
 # conventionally.
 
-def method_name
+
+def mult_add(a, b)
+  (a+b)*2
 end
 
+puts mult_add(a,b)
+
 Diagnostic.defined_method = # call your method here
 
```

Comment:
looking for something like this: Diagnostic.defined_method = add_and_multiply(3, 4)